docker-compose.yml

```diff
@@ -1,36 +1,43 @@
-version: "3"
+version: '3'
+
 services:
   jellystat-db:
-    image: postgres
+    image: postgres:latest
+    container_name: jellystat-db
+    restart: unless-stopped
     environment:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: mypassword
     volumes:
       - postgres-data:/var/lib/postgresql/data
+
   jellystat:
-    image: jellystat
+    image: cyfershepard/jellystat:latest
+    container_name: jellystat
+    restart: unless-stopped
     environment:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: mypassword
       POSTGRES_IP: jellystat-db
       POSTGRES_PORT: 5432
       JWT_SECRET: "my-secret-jwt-key"
-      TZ: Africa/Johannesburg
+      TZ: mytimezone # timezone (ex: Europe/Paris)
     volumes:
       - jellystat-backup-data:/app/backend/backup-data
     ports:
       - "3000:3000"
     depends_on:
       - jellystat-db
-    restart: unless-stopped
+      
 networks:
   default:
+
+volumes:
+  postgres-data:
+  jellystat-backup-data:
+
 logging:
```

Comment:
Does this I really relevant to keep?

Reply:
Hey, yea it is, without the postgres one, the data gets wiped whenever the container is restarted, and the jellystat one is needed to perform backups